test/abstractarray.jl

```diff
@@ -1286,24 +1286,24 @@ end
     ax=axes(a,1)
     @test a==a[ax]
     a=reshape(collect(1:100),(10,10))
-    @test a[:,1]==a[ax,1]
+    @test a[2:5]==2:5
     a=collect(1:10)
     ax=Base.OneTo(8)
     @test a[1:8]==a[ax]
     a=reshape(collect(1:100),(10,10))
     @test a[1:8,1]==a[ax,1]
     ax=UnitRange(1,10)
-    @test a[ax]==collect(1:10)
-    @test a[ax,1]==collect(1:10)
+    @test a[ax]==1:10
+    @test a[ax,1]==1:10
```

Comment:
This test and similar tests below are not using linear indexing. You only need to test for indexing with `ax`, no need to test for indexing with `ax, 1`.

Reply:
Changes added.